src/bin/ci/testConformanceLocal.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -e
+# import in shellcheck / CI / IntelliJ compatible ways
+# shellcheck source=/dev/null
+source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
+
+cromwell::build::setup_everyone_environment
```

Comment:
`everyone_environment`?

Reply:
I needed an word to describe "this does not need private information from Vault because it doesn't access the cloud and can run on any person's laptop". The term "public" seemed very overloaded already and "everyone" was chosen. Open to other suggestions.

Its sibling is called `cromwell::build::setup_secure_environment` if that should be renamed too.